src/microbe_stage/CompoundBag.cs

```diff
@@ -12,21 +12,16 @@ public class CompoundBag : ICompoundStorage
 {
     private readonly HashSet<Compound> usefulCompounds = new();
 
-    /// <summary>
-    ///   Creates a new bag
-    /// </summary>
-    /// <param name="capacity">Specifies the initial capacity of the compound bag</param>
-    public CompoundBag(float capacity)
-    {
-        Capacity = capacity;
-    }
+    [JsonProperty]
+    private float nominalCapacity;
```

Comment:
I don't see why this would need to be a field. Can't this be a property? That avoid having a "useless" manually written setter for the nominal capacity.